### ISSUE TITLE
try_get_value! refers to a private module

### ISSUE THREAD

**Issue by zsiciarz** [Keats/tera#94]
As I mentioned in [the reddit thread](https://www.reddit.com/r/rust/comments/5ijo5l/24_days_of_rust_tera/), the `try_get_value!` macro used in custom filters doesn't work outside the `tera` crate itself. This is due to this line in the macro expansion:

```rust
 return Err(::errors::TeraError::FilterIncorrectArgType(
```

but the `errors` module is private within `tera`. Here's a minimal example that fails to compile:
```rust
#[macro_use]
extern crate tera;
extern crate serde_json;

use std::collections::HashMap;
use tera::{TeraResult, Value};

pub fn dummy_filter(value: Value, _: HashMap<String, Value>) -> TeraResult<Value> {
    let _ = try_get_value!("dummy", "value", String, value);
    Ok(value)
}

fn main() {}
```

And the error is:
```
error[E0433]: failed to resolve. Maybe a missing `extern crate errors;`?
 --> src/tera.rs:9:13
  |
9 |     let _ = try_get_value!("dummy", "value", String, value);
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Maybe a missing `extern crate errors;`?
  |
  = note: this error originates in a macro outside of the current crate
```

**Comment by Keats** [comment]
Ah thanks for that!
I pushed a fix on the 0.5 branch that will (hopefully) be released monday so I'll close your PR as I don't plan another 0.4 release.

Thanks again for the report and the article!

**Comment by zsiciarz** [comment]
Wow, that was fast! Thanks for the heads up, looking forward to 0.5.
